infra/modules/azure_function_app/function_app.tf

```
@@ -50,8 +50,16 @@ resource "azurerm_linux_function_app" "this" {
     length(local.function_app.name) > 32 && !(contains(keys(var.app_settings), "AzureFunctionsWebHost__hostid")) ? { AzureFunctionsWebHost__hostid = "production" } : {},
     var.app_settings,
     local.application_insights.enable ? {
+      # SDK AI Sampling (for dependencies)
       # https://docs.microsoft.com/en-us/azure/azure-monitor/app/sampling
-      APPINSIGHTS_SAMPLING_PERCENTAGE = var.application_insights_sampling_percentage
+      APPINSIGHTS_SAMPLING_PERCENTAGE = var.application_insights_sampling_percentage,
+
+      # Runtime AI Sampling (for requests, traces, metrics, etc..)
```

```suggestion
      # Azure Function Host (runtime) AI Sampling (for requests, traces, metrics, etc..)
```

For the casual reader 'Runtime' may refer to some 'runtime' behavior.